packages/eslint-plugin-pf-codemods/src/rules/v6/componentGroupsInvalidObjectPropsRenameToMissingPageProps/component-groups-invalidObjectProps-rename-to-missingPageProps.test.ts

```diff
@@ -0,0 +1,95 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./component-groups-invalidObjectProps-rename-to-missingPageProps";
+
+const message = `InvalidObjectProps has been renamed to MissingPageProps.`;
+
+ruleTester.run(
+  "component-groups-invalidObjectProps-rename-to-missingPageProps",
+  rule,
+  {
+    valid: [
+      // missing import
+      {
+        code: `const props: InvalidObjectProps;`,
+      },
+      // import from wrong package
+      {
+        code: `import { InvalidObjectProps } from '@patternfly/react-core';`,
+      },
+    ],
+    invalid: [
+      {
+        code: `import { InvalidObjectProps } from '@patternfly/react-component-groups';
```

Comment:
Maybe a valid test for importing something else from the correct package, and an invalid test that imports InvalidObjectProps and something else to show that only InvalidObjectProps gets touched/flagged?